libs/boost/Makefile

```diff
@@ -11,13 +11,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=boost
-PKG_VERSION:=1.80.0
-PKG_SOURCE_VERSION:=1_80_0
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_VERSION:=1.81.0
+PKG_SOURCE_VERSION:=1_81_0
+PKG_RELEASE:=4
```

Comment:
It should be reset to `1` for version bumping.

Reply:
I was following what the bot said to do.

Reply:
Updated in 6c35386

Reply:
Yup, sadly the bot doesn't seem to work properly ;(